tests/test_kernels/default_settings/Cargo.toml

```
@@ -6,5 +6,5 @@ edition = "2018"
 
 [dependencies]
 bootloader = { path = "../../.." }
-x86_64 = "0.13.2"
+x86_64 = { version = "0.13.2", optional = true, default-features = false, features = ["instructions", "inline_asm"] }
```

```suggestion
x86_64 = { version = "0.13.2", default-features = false, features = ["instructions", "inline_asm"] }
```
This wasn't optional before. Same for the other test kernel.

---

Good catch, thanks a lot!